pkg/virt-controller/watch/network_test.go

```diff
@@ -19,13 +19,14 @@
 package watch
 
 import (
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```

Comment:
nit: This import should remain at this position, along with other exeternal API's